spinalcordtoolbox/image.py

```diff
@@ -516,19 +504,16 @@ def getNonZeroCoordinates(self, sorting=None, reverse_coord=False, coordValue=Fa
         if self.dim[2] == 1:
             n_dim = 2
 
-        try:
-            if n_dim == 3:
+        if n_dim == 3:
```

Comment:
hum, i'm not sure i follow the logic here (we lost the try/except for the n_dim=3 case), but the original code probably had some flaws... so maybe addressing it in this PR is out of scope. Just mentioning in case there is a problem that needs to be addressed at some point

Reply:
I did this wherever I saw the following pattern:

```
try:
    something()
except Exception as e:
   sct.printv("some error message)
```

The reason being that an error is detected, but the exception is ignored without an explicit `pass` or message saying that it's normal for the exception to be ignored. This leads me to believe that it's possibly a mistake and that the code should raise the exception to the parent.

Reply:
ok got it